content/tutorial/tutorial.md

```diff
@@ -166,9 +166,9 @@ return React.createElement('div', {className: 'shopping-list'},
 
 Əgər sizə maraqlıdırsa, `createElement()` [API arayışında](/docs/react-api.html#createelement) daha detallı izah edilir. Biz bu dərslikdə `createElement()` əvəzinə JSX-dən istifadə etməyə davam edəcəyik.
 
-JSX, Javascriptin bütün gücü ilə gəlir. Siz *hər hansı* Javascript ifadəsini JSX-də fiqurlu mötərizənin içərisindən çağıra bilərsiniz. Hər React elementi bir sadə Javascript obyekti obyektidir. Siz bu obyekti dəyişənə təyin edə bilər və ya applikasiyanızda funskiyalara və s. göndərə bilərsiniz.
+JSX, Javascriptin bütün gücü ilə gəlir. Siz *hər hansı* Javascript ifadəsini JSX-də fiqurlu mötərizənin içərisindən çağıra bilərsiniz. Hər React elementi bir sadə Javascript obyektidir. Siz bu obyekti dəyişənə təyin edə bilər və ya applikasiyanızda funksiyalara və s. göndərə bilərsiniz.
 
-Yuxarıdakı nümunədə, `ShoppingList` komponenti yalnız hazır qurulmuş DOM komponentlərini (`<div />` və `<li />`) render edir. Amma siz başqa xüsusi komponentləri belə biləşdirə bilərsiniz. Məsələn, siz yuxarıdakı alqı satqı siyahısını `<ShoppingList />` kimi işlədə bilərsiniz. Hər React komponenti inkapsulasiya olunub deyə siz bu komponentləri bir-birindən asılı olmayacaq şəkildə istifadə edə bilərsiniz. Bu xüsusiyyət sadə komponentlərdən kompleks UI-lar yaratmağa icazə verir.
+Yuxarıdakı nümunədə, `ShoppingList` komponenti yalnız hazır qurulmuş DOM komponentlərini (`<div />` və `<li />`) render edir. Lakin, siz həmçinin öz komponentlərinizi də yarada bilərsiniz. Məsələn, siz yuxarıdakı alqı satqı siyahısını `<ShoppingList />` kimi işlədə bilərsiniz. Hər React komponenti inkapsulasiya olunub deyə siz bu komponentləri bir-birindən asılı olmayacaq şəkildə istifadə edə bilərsiniz. Bu xüsusiyyət sadə komponentlərdən mürəkkəb UI-lər yaratmağa icazə verir.
```

Comment:
> UI-lər

Before the project started, we have decided to always pronounce abbreviations of terms in English. Example, API is [ey pi ay], not [a p i]. Same goes for UI. I will update the glossary to show the pronunciation of UI. In the meantime, please revert this change.

Reply:
Both of these end with the interface [interfeys]. So it actually requires "lər" suffix. Why do we focus on Abbreviations ? Is there a particular reason for this?

Reply:
Yes, but when you pronounce UI, you do not say [yu i], you say [yu ay] -- you pronounce the letters. As a result, UI-lər ends up being pronounced as [ui ayLƏR], which makes it very unnatural in Azerbaijani language.

I am focusing on abbreviations because these abbreviations do not have translations in Azerbaijani. For example, we can translate "user interface" as "istifadəçi interfeysi" but we don't translate UI as İİ. If we did, you would be right: "İİ-lər" would be more appropriate because the abbreviation letters are derived from an Azerbaijani word - istifadəçi interfeysi. Because UI is an abbreviation of an English word (user interface), we pronounce the letters in English. Yes, people typically pronounce UI as both [yu ay] and [u i] in our language; so, we had to make a decision that is consistent across the entire website. We chose to pronounce all English abbreviations in English because the reader will always know to pronounce ALL abbreviations in English when reading the docs instead of trying to figure each word individually.

Reply:
I got your point. But given the fact that the vast majority of our people do not speak English well. We must make the content as user-friendly and readable as possible. We can even discuss this in a fb group.